test/integration/006_simple_dependency_test/models/disabled_one.sql

```diff
@@ -0,0 +1 @@
+{{config(enabled=False)}}
```

Comment:
this is dumb, but can you just add `select 1` here? I can foresee this getting muddied up with logic that passes over empty models